packages/back-end/src/routers/archetype/archetype.controller.ts

```diff
@@ -147,6 +149,7 @@ type CreateArchetypeRequest = AuthRequest<{
   owner: string;
   isPublic: boolean;
   attributes: string;
+  projects?: string[];
```

Comment:
Also need to update all of the permission functions related to archetypes to be project scoped.